web/src/pages/oss.js

```diff
@@ -18,12 +21,24 @@ export default function Oss() {
     .filter((v, i, a) => a.indexOf(v) === i) // remove duplicates
     .sort((a, b) => a.localeCompare(b)) // alphabetical
   const maintainerPills = ['All'].concat(maintainerTags)
-  
+
+  React.useEffect(async () => {
+    const reposResponse = await octokit.request("GET /orgs/{org}/repos", {
+      org: "supabase",
+      type: "public",
+      sort: "updated",
```

Comment:
Surely this wants to be the six biggest libs, or have it be paginated.

Reply:
Let's just make it all of them. Keep it simple for now

Reply:
Sounds good, will do so in the updated [pull request](https://github.com/supabase/supabase/pull/5693).